help/deactivate-or-reactivate-a-user.md

```diff
@@ -19,7 +19,9 @@ When you deactivate a user:
   user will be disabled.
 
 * Other users will be able to see that the user has been deactivated (e.g., on
-  their [user card](/help/user-cards)).
+  their [user card](/help/user-cards)). In sidebars and elsewhere, a user's
+  [status](/help/status-and-availability) will render as a banned dot
```

Comment:
I don't think `banned dot` is a term we want to use -- I think `a user's [availability]() will be replaced with a deactivated icon (<span...>)` would be right/

(We reserve the word "status" for the emoji thing, not presence/availability)

Reply:
Yeah, @timabbott 's proposal works for  me.

Reply:
Cool. Updated the copy accordingly. Thank you!